test/ArrayField_test.js

```diff
@@ -38,6 +41,31 @@ describe("ArrayField", () => {
       expect(legend.id).eql("root__title");
     });
 
+    it("should render a description", () => {
+      const {node} = createFormComponent({schema});
+
+      const description = node.querySelector("fieldset > div");
+
+      expect(description.textContent).eql("my description");
+      expect(description.id).eql("root__description");
+    });
+
+    it("should render a customized title", () => {
+      const CustomTitleField = ({title}) => <div id="custom">{title}</div>;
+
+      const {node} = createFormComponent({schema, TitleField: CustomTitleField});
+      expect(node.querySelector("fieldset > #custom").textContent)
+      .to.eql("my list");
+    });
+
+    it("should render a customized description", () => {
+      const CustomDescriptionField = ({description}) => <div id="custom">{description}</div>;
+
+      const {node} = createFormComponent({schema, DescriptionField: CustomDescriptionField});
```

Comment:
I'd rather setup this test with the officially supported api, which is to pass a `fields` prop (the one used is kept for BC but is not planned for long term support).